requirements.txt

```diff
@@ -9,10 +9,8 @@
 # Needed to run the Linux service.
 daemonize==2.4.7
 
-# Python Jira api.
-# Our fork provides a few fixes that are not part of the official API. Unfortunately
-# contributing back is going to be difficult because Python 2 is not supported anymore.
-git+https://github.com/shotgunsoftware/jira.git@2.0.0.sg.3#egg=jira
```

Comment:
Should we plan to archive this repository then?

Reply:
It should be good. However, I'm not sure if there are users that use it for their current versions.

Reply:
I am saying to delete it. We can start with archive which makes it read-only and make it clear to anyone that it's legacy.

Reply:
Will include this in our clean-up duties.